k-distribution/include/builtin/rat.k

```diff
@@ -0,0 +1,113 @@
+module RAT-SYNTAX
+  imports INT-SYNTAX
+  imports BOOL
+
+  syntax Rat
+
+  syntax Rat ::= "<" Int "," Int ">Rat" [format(%2 /Rat %4)]
```

Comment:
let's move this constructor into the RAT module, since it's not part of the public signature.